charts/airflow/values.yaml

```diff
@@ -197,6 +197,9 @@ airflow:
     ## this does NOT give root permissions to Pods, only the "root" group
     fsGroup: 0
 
+  ## default priorityClassName for airflow Pods (is overridden by pod-specific values)
+  defaultPriorityClassName: ""
```

Comment:
To keep things consistent let's use the same formatting as the other docstrings:

```yaml
  ## default priorityClassName for airflow Pods (is overridden by pod-specific values)
  ## - docs for priorityClassName:
  ##   https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
  ##
  defaultPriorityClassName: ""
```